docs/tutorials/validate-results/collect-simulation-parameters.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: default
+title: Collect parameters for simulation
+grand_parent: /tutorials
+parent: /tutorials/validate-results
+nav_order: 1
+nav_exclude: true
+has_toc: false
+---
+
+
+
+# Validate results
+{: .no_toc }
+
+Follow this procedure to validate the state configuration, state populations and state transition rates determined from an experimental smFRET video analysis.
+
+{% include tutorial_head.html %}
+
+In this step, a maximum number of parameters are collected or calculated from the previous experimental data analysis.
+
+1. TOC
+{:toc}
+
+---
+
+{% comment %}
+1. Use the **files** exported during your previous analysis to collect or calculate <u>experimental parameters</u>:  
+     
+   &#9745; number of molecules  
+   &#9745; summed fluorescence <u>intensities</u> (corrected donor + acceptor intensities)  (in 
+   [trace files](../../output-files/txt-processed-traces.html))  
+   &#9745; average channel-specific <u>background</u> intensities  (in 
+   [parameters files](../../output-files/log-processing-parameters.html))  
+   &#9745; <u>FRET states</u>  
+   &#9745; state <u>transition rates</u>  
+   
+1. Use the experimental parameters and your **setup characteristics** in 
+[Simulation](../../simulation) to <u>generate</u> synthetic intensity-time traces.  
+      
+	**Note:** *To simulate kinetic heterogeneity, use a number of degenerated FRET states equal to the number of exponential components*.
```

Comment:
I would like to cite Schmid et al. DOI: 10.1063/1.5006604

Reply:
If it's about simulating kinetic heterogeneity, I'd rather put the reference in simulation/functionalities/simulate-kintetic-heterogeneity.md

Reply:
its is about the definition of degenerated FRET states and kinetic heterogeneity